src/compressor.h

```diff
@@ -10,8 +10,9 @@
 #include <script/script.h>
 #include <serialize.h>
 #include <span.h>
+#include <prevector.h>
 
-bool CompressScript(const CScript& script, std::vector<unsigned char> &out);
+bool CompressScript(const CScript& script, prevector<33, unsigned char> &out);
```

Comment:
nit: could you fix the reference whitespace? See:
https://github.com/bitcoin/bitcoin/blob/master/contrib/devtools/README.md#clang-format-diffpy